### ISSUE TITLE
Create banner warning users that v2 website is not finished

### ISSUE THREAD

**Issue by maxachis** [Police-Data-Accessibility-Project/data-sources-app#534]
### Context

At the moment, users could, however improbably, come to our new v2 website and think it's our main website. Maybe we should let them know that it's not ready?

### Requirements

* Create a toast v2 web page with the following warning:

> This is the v2 version of the PDAP search, and is still under development! Data here is unreliable, and created accounts are deleted on a regular basis. For the current version, head to https://data-sources.pdap.io

### Tests

* No tests needed

### Docs

* No documentation needed

### Open questions

* None

**Comment by josh-chamberlain** [comment]
I updated the copy a bit. I think we can use the lovely `toast` infrastructure for this.

**Comment by joshuagraber** [comment]
@maxachis @josh-chamberlain I missed this when created, but yeah, this should be easily do-able. However, since we're nearly ready to release, is it still applicable? We could, for instance, just rip out the `data-sources-v2` client, which would also solve the problem.

**Comment by josh-chamberlain** [comment]
@joshuagraber yes, it's probably safe just to remove the client now that it's in `dev` and releasing shortly.

**Comment by maxachis** [comment]
Not needed! We shall close.
